links.html

```diff
@@ -13,7 +13,7 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
     <link rel="icon" href="favicon.ico" type="image/x-icon">
-    <link href="https://fonts.googleapis.com/css?family=Domine/Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Domine%7CRoboto&display=swap" rel="stylesheet">
```

Comment:
Don't replace the `|` symbol with `/`. Instead use the url-encoded value for `|`: %7C